bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/BeneficiaryRow.java

```diff
@@ -189,93 +181,135 @@ public String toString() {
 		DML_IND,
 
 		/**
-		 * Type: <code>CHAR</code>, max chars: 15.
+		 * Type: <code>CHAR</code>, max chars: 15. See <a href=
+		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/bene_id.txt">
+		 * CCW Data Dictionary: BENE_ID</a>, though note that this instance of +
```

Comment:
I think this comment got a little malformed with the + in there. Also should specify that this instance of the field is **not** encrypted.